kornia/geometry/transform/pyramid.py

```diff
@@ -350,3 +350,49 @@ def build_pyramid(
         pyramid.append(img_down)
 
     return pyramid
+
+
+def build_laplacian_pyramid(
+    input: torch.Tensor, max_level: int, border_type: str = 'reflect', align_corners: bool = False
+) -> List[torch.Tensor]:
+    r"""Construct the Laplacian pyramid for an image.
+
+    .. image:: _static/img/build_pyramid.png
+
+    The function constructs a vector of images and builds the Laplacian pyramid
+    by recursively computing the difference after applying
+    pyrUp to the adjacent layer in it's Gaussian pyramid.
+
+    Args:
+        input : the tensor to be used to construct the pyramid.
```

Comment:
```suggestion
        input: the tensor to be used to construct the pyramid with shape :math:`(B, C, H, W)`.
```
for free functions we tend to not use the `Shape` directive